ui-icons/scripts/build.js

```diff
@@ -152,10 +152,10 @@ for (const [name, currentData] of Object.entries(representation)) {
 
     return [
       `IconReplaceName${variant}Props`,
-      `interface IconReplaceName${variant}Props { size?: ${sizeStr}; variant?: "${variant}"; title?: string; };`,
+      `interface IconReplaceName${variant}Props { size?: ${sizeStr}; variant?: "${variant}"; title?: string; iconColor?: string; className?: string; };`,
```

Comment:
A friendly suggestion, maybe the variable name `iconFillColor` is better, since the variable applies a colour that `fills` the interior of the icon? :)

Reply:
I have a doubt... fill is usually used as an SVG attribute, here it's not exactly the same concept

Reply:
For most of the icons, it doesn't fill the "interior" of the icons but only changes the stroke color of it. As we don't have 2 distinct props to change to color of the stroke or the "interior", I think it's fine to keep this prop named "iconColor" (maybe just `color` would have been fine too)